test/unit/test_lif_cell_group.cpp

```diff
@@ -158,7 +159,7 @@ TEST(lif_cell_group, spikes) {
     path_recipe recipe(2, 1000, 0.1);
 
     hw::node_info nd;
-    nd.num_cpu_cores = threading::num_threads();
+    nd.num_cpu_cores = arb::thread_count();
```

Comment:
To minimise the surface area of this PR, lI would avoid renamings like`num_threads` to `thread_count`.
Also, `num_threads` is a better variable name.

Reply:
You know, I hate those `num_`names. But yeah, it doesn't matter what it is called. It's not replacing the underlying call to the thread implementation, but wrapping it.

Reply:
I know you don't like them, but that is the name we have, and wrapping it with a different name is renaming it ;)